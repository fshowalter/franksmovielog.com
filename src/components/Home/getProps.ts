import { loadExcerptHtml, mostRecentReviews } from "src/api/reviews";
import { getStillImageProps } from "src/api/stills";

import type { Props } from "./Home";
import { StillImageConfig } from "./HomeListItem";

export async function getProps(): Promise<Props> {
  const titles = await mostRecentReviews(11);

  const values = await Promise.all(
    titles.map(async (review) => {
      return await loadExcerptHtml(review);
    }),
  );

  return {
    values: await Promise.all(
      values.map(async (value) => {
        return {
          ...value,
          stillImageProps: await getStillImageProps(
            value.slug,
            StillImageConfig,
          ),
        };
      }),
    ),
  };
}
