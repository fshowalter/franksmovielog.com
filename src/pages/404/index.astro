---
import { Article } from "~/components/article/Article";
import { getProps } from "~/components/article/getProps";
import Layout from "~/layouts/Layout.astro";

const slug = "not-found";

const deck = '"Dick Laurent is dead."';

const props = await getProps({
  deck,
  slug,
});
---

<Layout
  canonical={false}
  meta={{ description: deck }}
  openGraph={false}
  title="404: Not Found"
>
  <Article {...props} />
</Layout>
