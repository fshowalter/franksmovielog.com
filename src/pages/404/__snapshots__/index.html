<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta content="width=device-width" name="viewport" />
    <link href="/favicon.ico" rel="icon" sizes="48x48" />
    <link href="/favicon.svg" rel="icon" sizes="any" type="image/svg+xml" />
    <link href="/apple-touch-icon.png" rel="apple-touch-icon" />
    <link href="/site.webmanifest" rel="manifest" />
    <link href="/sitemap-index.xml" rel="sitemap" />
    <meta content="&#34;Dick Laurent is dead.&#34;" name="description" />
    <meta content="rgba(0,0,0,.85)" name="theme-color" />
    <link
      href="https://www.franksmovielog.com/feed.xml"
      rel="alternate"
      title="Frank's Movie Log"
      type="application/rss+xml"
    />
    <title>404: Not Found - Frank&#39;s Movie Log</title>
  </head>
  <body>
    <link
      rel="preload"
      as="image"
      imagesrcset="/_image/?href=%2Fcontent%2Fassets%2Fbackdrops%2Fnot-found.png%3ForigWidth%3D2400%26origHeight%3D1350%26origFormat%3Dpng&amp;w=720&amp;h=405&amp;q=80&amp;f=avif 720w, /_image/?href=%2Fcontent%2Fassets%2Fbackdrops%2Fnot-found.png%3ForigWidth%3D2400%26origHeight%3D1350%26origFormat%3Dpng&amp;w=1200&amp;h=675&amp;q=80&amp;f=avif 1200w, /_image/?href=%2Fcontent%2Fassets%2Fbackdrops%2Fnot-found.png%3ForigWidth%3D2400%26origHeight%3D1350%26origFormat%3Dpng&amp;w=1920&amp;h=1080&amp;q=80&amp;f=avif 1920w, /_image/?href=%2Fcontent%2Fassets%2Fbackdrops%2Fnot-found.png%3ForigWidth%3D2400%26origHeight%3D1350%26origFormat%3Dpng&amp;w=2400&amp;h=1350&amp;q=80&amp;f=avif 2400w"
      imagesizes="100vw"
      fetchpriority="high"
    />
    <div>
      <a
        class="absolute left-1/2 top-0.5 z-50 mx-auto bg-subtle px-6 py-2 text-center text-accent [transform:translate(-50%,calc(-100%_-_2px))] focus:[transform:translate(-50%,0%)]"
        href="#content"
        >Skip to content</a
      >
      <div class="flex min-h-full w-full flex-col bg-default">
        <header
          class="z-20 flex w-full items-center justify-between px-container py-4 tablet:p-6 desktop:inset-x-0 desktop:z-40 desktop:flex-row desktop:flex-wrap desktop:px-16 desktop:py-8 desktop:text-left"
          style="
            background-image: linear-gradient(
              to bottom,
              rgba(0, 0, 0, 0.85),
              transparent 95%
            );
            color: #fff;
            position: absolute;
          "
        >
          <div class="flex flex-col">
            <div
              class="whitespace-nowrap font-normal leading-8"
              style="font-size: 1.5625rem"
            >
              <a href="/">Frank&#x27;s Movie Log</a>
            </div>
            <p class="w-full pl-px text-sm italic leading-4 opacity-85">
              by Frank Showalter
            </p>
          </div>
          <div class="flex items-center">
            <nav class="hidden w-full desktop:block desktop:w-auto">
              <ul class="flex flex-wrap justify-start gap-x-6 text-xl">
                <li class="block whitespace-nowrap tracking-serif-wide">
                  <a
                    class="text-inherit transition-all duration-500 ease-in-out hover:bg-[rgba(0,0,0,.50)]"
                    href="/"
                    style="text-shadow: 1px 1px 2px black"
                    >Home</a
                  >
                </li>
                <li class="block whitespace-nowrap tracking-serif-wide">
                  <a
                    class="text-inherit transition-all duration-500 ease-in-out hover:bg-[rgba(0,0,0,.50)]"
                    href="/how-i-grade/"
                    style="text-shadow: 1px 1px 2px black"
                    >How I Grade</a
                  >
                </li>
                <li class="block whitespace-nowrap tracking-serif-wide">
                  <a
                    class="text-inherit transition-all duration-500 ease-in-out hover:bg-[rgba(0,0,0,.50)]"
                    href="/reviews/"
                    style="text-shadow: 1px 1px 2px black"
                    >Reviews</a
                  >
                </li>
                <li class="block whitespace-nowrap tracking-serif-wide">
                  <a
                    class="text-inherit transition-all duration-500 ease-in-out hover:bg-[rgba(0,0,0,.50)]"
                    href="/viewings/"
                    style="text-shadow: 1px 1px 2px black"
                    >Viewing Log</a
                  >
                </li>
                <li class="block whitespace-nowrap tracking-serif-wide">
                  <a
                    class="text-inherit transition-all duration-500 ease-in-out hover:bg-[rgba(0,0,0,.50)]"
                    href="/cast-and-crew/"
                    style="text-shadow: 1px 1px 2px black"
                    >Cast &amp; Crew</a
                  >
                </li>
                <li class="block whitespace-nowrap tracking-serif-wide">
                  <a
                    class="text-inherit transition-all duration-500 ease-in-out hover:bg-[rgba(0,0,0,.50)]"
                    href="/collections/"
                    style="text-shadow: 1px 1px 2px black"
                    >Collections</a
                  >
                </li>
                <li class="block whitespace-nowrap tracking-serif-wide">
                  <a
                    class="text-inherit transition-all duration-500 ease-in-out hover:bg-[rgba(0,0,0,.50)]"
                    href="/watchlist/"
                    style="text-shadow: 1px 1px 2px black"
                    >Watchlist</a
                  >
                </li>
              </ul>
            </nav>
            <div class="z-[1000]">
              <button
                aria-keyshortcuts="Control+K"
                aria-label="Search"
                class="flex size-10 items-center justify-center overflow-hidden text-sm leading-6 ring-default desktop:ml-6"
                data-open-modal="true"
                disabled=""
                title="Search: Control+K"
                type="button"
              >
                <svg
                  class="size-7"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="1.5"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="m21 21-5.197-5.197m0 0A7.5 7.5 0 1 0 5.196 5.196a7.5 7.5 0 0 0 10.607 10.607Z"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  ></path>
                </svg>
              </button>
            </div>
            <input class="hidden" id="mobile-nav" type="checkbox" /><label
              class="hamburger-icon desktop:hidden"
              for="mobile-nav"
              ><span class="hamburger-icon-bars" style="background: #fff"></span
            ></label>
            <ul
              class="hamburger-menu flex flex-col items-start gap-y-5 text-left text-inverse desktop:hidden"
            >
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a href="/">Home</a>
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a href="/how-i-grade/">How I Grade</a>
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a href="/reviews/">Reviews</a>
                <ol class="mt-4">
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wider text-inverse-subtle last:mb-0"
                  >
                    <a href="/reviews/overrated/">Overrated</a>
                  </li>
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wider text-inverse-subtle last:mb-0"
                  >
                    <a href="/reviews/underrated/">Underrated</a>
                  </li>
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wider text-inverse-subtle last:mb-0"
                  >
                    <a href="/reviews/underseen/">Underseen</a>
                  </li>
                </ol>
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a href="/viewings/">Viewing Log</a>
                <ol class="mt-4">
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wider text-inverse-subtle last:mb-0"
                  >
                    <a href="/viewings/stats/">Stats</a>
                  </li>
                </ol>
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a href="/cast-and-crew/">Cast &amp; Crew</a>
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a href="/collections/">Collections</a>
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a href="/watchlist/">Watchlist</a>
                <ol class="mt-4">
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wider text-inverse-subtle last:mb-0"
                  >
                    <a href="/watchlist/progress/">Progress</a>
                  </li>
                </ol>
              </li>
            </ul>
          </div>
        </header>
        <main class="grow" id="content">
          <article>
            <header
              class="min-h-[90vh] max-h-[1350px] relative flex w-full flex-col content-start items-center justify-end gap-6 bg-[#2A2B2A] pb-8 pt-40 text-inverse tablet:pb-10 tablet:pt-40 desktop:pb-16 desktop:pt-40"
            >
              <img
                class="absolute inset-0 size-full object-cover object-top"
                src="/_image/?href=%2Fcontent%2Fassets%2Fbackdrops%2Fnot-found.png%3ForigWidth%3D2400%26origHeight%3D1350%26origFormat%3Dpng&amp;w=2400&amp;h=1350&amp;q=80&amp;f=avif"
                srcset="
                  /_image/?href=%2Fcontent%2Fassets%2Fbackdrops%2Fnot-found.png%3ForigWidth%3D2400%26origHeight%3D1350%26origFormat%3Dpng&amp;w=720&amp;h=405&amp;q=80&amp;f=avif    720w,
                  /_image/?href=%2Fcontent%2Fassets%2Fbackdrops%2Fnot-found.png%3ForigWidth%3D2400%26origHeight%3D1350%26origFormat%3Dpng&amp;w=1200&amp;h=675&amp;q=80&amp;f=avif  1200w,
                  /_image/?href=%2Fcontent%2Fassets%2Fbackdrops%2Fnot-found.png%3ForigWidth%3D2400%26origHeight%3D1350%26origFormat%3Dpng&amp;w=1920&amp;h=1080&amp;q=80&amp;f=avif 1920w,
                  /_image/?href=%2Fcontent%2Fassets%2Fbackdrops%2Fnot-found.png%3ForigWidth%3D2400%26origHeight%3D1350%26origFormat%3Dpng&amp;w=2400&amp;h=1350&amp;q=80&amp;f=avif 2400w
                "
                height="1350"
                sizes="100vw"
                width="2400"
                alt=""
                fetchpriority="high"
                loading="eager"
              />
              <div
                class="z-10 mx-auto flex w-full max-w-screen-max flex-col px-container"
              >
                <h1
                  class="font-sans text-2xl font-bold uppercase tracking-widest desktop:text-7xl"
                  data-pagefind-weight="10"
                >
                  404 Not Found
                </h1>
                <p
                  class="mt-1 text-base desktop:my-4 desktop:text-xl [text-shadow:1px_1px_2px_black]"
                >
                  &quot;Dick Laurent is dead.&quot;
                </p>
              </div>
            </header>
            <section class="flex flex-col items-center pb-32 pt-16">
              <div class="px-container">
                <div
                  class="rendered-markdown text-md/7 tracking-prose tablet:text-xl/8 max-w-prose"
                >
                  <p>
                    This is probably my fault. Are you looking for one of my old
                    reviews? I’m in the process of touching them up and
                    reposting them. In the mean time, you can find them in the
                    site’s
                    <a href="https://github.com/fshowalter/franksmovielog.com/"
                      >GitHub repo</a
                    >.
                  </p>
                  <p>
                    If you still can’t find what you need,
                    <a
                      href="https://github.com/fshowalter/franksmovielog.com/issues/new/choose"
                      >open an issue on GitHub</a
                    >
                    and I’ll try to help.
                  </p>
                  <p>My apologies, and thanks for stopping by.</p>
                </div>
              </div>
            </section>
          </article>
          <div class="bg-subtle pb-16 pt-6">
            <nav data-pagefind-ignore="true">
              <div
                class="relative mx-auto w-full max-w-screen-max px-container desktop:px-20"
              >
                <h2
                  class="py-10 font-sans text-xs font-semibold uppercase tracking-wide text-subtle"
                >
                  Recent<!-- -->
                  <a class="text-accent" href="/reviews/">Reviews</a>
                </h2>
                <ul
                  class="flex w-full flex-col gap-[3%] gap-y-[6vw] tablet:flex-row tablet:flex-wrap tablet:justify-between desktop:flex-nowrap"
                >
                  <li class="relative w-full tablet:w-[47%]">
                    <div class="block">
                      <img
                        src="/_image/?href=%2Fcontent%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=640&amp;h=360&amp;q=80&amp;f=avif"
                        srcset="
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=160&amp;h=90&amp;q=80&amp;f=avif    160w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=320&amp;h=180&amp;q=80&amp;f=avif   320w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=640&amp;h=360&amp;q=80&amp;f=avif   640w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fchamber-of-horrors-1966.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=1280&amp;h=720&amp;q=80&amp;f=avif 1280w
                        "
                        alt=""
                        decoding="async"
                        loading="lazy"
                        height="360"
                        sizes="(max-width: 767px) 84vw, (max-width: 1279px) calc((100vw - 96px) * 0.47), (max-width: 1695px) calc((100vw - 160px) * .22735), 350px"
                        width="640"
                        class="aspect-video h-auto w-full"
                      />
                    </div>
                    <div
                      class="flex flex-col bg-default px-6 pb-4 pt-6 desktop:pl-[12%] desktop:pr-[14%]"
                    >
                      <a
                        class="mb-3 block text-xl font-medium leading-6 text-default before:absolute before:inset-x-0 before:top-0 before:aspect-video before:bg-[#fff] before:opacity-15 hover:text-accent hover:before:opacity-0"
                        href="/reviews/chamber-of-horrors-1966/"
                        >Chamber of Horrors<!-- --> <span
                          class="text-sm font-normal leading-none text-muted"
                          >1966</span
                        ></a
                      ><picture
                        ><source
                          media="(prefers-color-scheme: dark)"
                          srcset="/svg/2-stars-dark.svg" />
                        <img
                          alt="D+: 2 stars (out of 5)"
                          class="mb-4"
                          height="18"
                          src="/svg/2-stars.svg"
                          width="90"
                      /></picture>
                      <div
                        class="rendered-markdown mb-8 text-base leading-[1.6] tracking-prose text-muted"
                      >
                        <p>
                          Warner Bros. shot <em>Chamber of Horrors</em> as a
                          television movie, intended as a series pilot. When the
                          network deemed the finished product too violent for
                          television, the studio added some gimmicks and
                          released it as a theatrical feature. The result looks
                          good but underwhelms.
                          <a
                            class="!no-underline uppercase whitespace-nowrap font-normal font-sans text-accent text-xs leading-none hover:!underline"
                            href="/reviews/chamber-of-horrors-1966/"
                            >Continue reading...</a
                          >
                        </p>
                      </div>
                      <div
                        class="font-sans text-xxs font-light leading-4 tracking-wider text-subtle desktop:tracking-wide"
                      >
                        <span>Crime</span
                        ><span>
                          |
                          <!-- -->Horror</span
                        ><span>
                          |
                          <!-- -->Thriller</span
                        >
                      </div>
                    </div>
                  </li>
                  <li class="relative w-full tablet:w-[47%]">
                    <div class="block">
                      <img
                        src="/_image/?href=%2Fcontent%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=640&amp;h=360&amp;q=80&amp;f=avif"
                        srcset="
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=160&amp;h=90&amp;q=80&amp;f=avif    160w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=320&amp;h=180&amp;q=80&amp;f=avif   320w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=640&amp;h=360&amp;q=80&amp;f=avif   640w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Ftwo-for-the-seesaw-1962.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=1280&amp;h=720&amp;q=80&amp;f=avif 1280w
                        "
                        alt=""
                        decoding="async"
                        loading="lazy"
                        height="360"
                        sizes="(max-width: 767px) 84vw, (max-width: 1279px) calc((100vw - 96px) * 0.47), (max-width: 1695px) calc((100vw - 160px) * .22735), 350px"
                        width="640"
                        class="aspect-video h-auto w-full"
                      />
                    </div>
                    <div
                      class="flex flex-col bg-default px-6 pb-4 pt-6 desktop:pl-[12%] desktop:pr-[14%]"
                    >
                      <a
                        class="mb-3 block text-xl font-medium leading-6 text-default before:absolute before:inset-x-0 before:top-0 before:aspect-video before:bg-[#fff] before:opacity-15 hover:text-accent hover:before:opacity-0"
                        href="/reviews/two-for-the-seesaw-1962/"
                        >Two for the Seesaw<!-- --> <span
                          class="text-sm font-normal leading-none text-muted"
                          >1962</span
                        ></a
                      ><picture
                        ><source
                          media="(prefers-color-scheme: dark)"
                          srcset="/svg/3-stars-dark.svg" />
                        <img
                          alt="C+: 3 stars (out of 5)"
                          class="mb-4"
                          height="18"
                          src="/svg/3-stars.svg"
                          width="90"
                      /></picture>
                      <div
                        class="rendered-markdown mb-8 text-base leading-[1.6] tracking-prose text-muted"
                      >
                        <p>
                          In <em>Two for the Seesaw</em>, Robert Mitchum plays a
                          middle-aged, midwestern lawyer who—after his wife of
                          twelve years asks for a divorce—flees his cushy job
                          and pampered lifestyle for New York City. There, he
                          meets a late-twenty-something dancer played by Shirley
                          MacLaine. A romance blooms.
                          <strong>Spoilers follow</strong>.
                          <a
                            class="!no-underline uppercase whitespace-nowrap font-normal font-sans text-accent text-xs leading-none hover:!underline"
                            href="/reviews/two-for-the-seesaw-1962/"
                            >Continue reading...</a
                          >
                        </p>
                      </div>
                      <div
                        class="font-sans text-xxs font-light leading-4 tracking-wider text-subtle desktop:tracking-wide"
                      >
                        <span>Drama</span
                        ><span>
                          |
                          <!-- -->Romance</span
                        >
                      </div>
                    </div>
                  </li>
                  <li class="relative w-full tablet:w-[47%]">
                    <div class="block">
                      <img
                        src="/_image/?href=%2Fcontent%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=640&amp;h=360&amp;q=80&amp;f=avif"
                        srcset="
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=160&amp;h=90&amp;q=80&amp;f=avif    160w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=320&amp;h=180&amp;q=80&amp;f=avif   320w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=640&amp;h=360&amp;q=80&amp;f=avif   640w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fthe-man-with-nine-lives-1940.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=1280&amp;h=720&amp;q=80&amp;f=avif 1280w
                        "
                        alt=""
                        decoding="async"
                        loading="lazy"
                        height="360"
                        sizes="(max-width: 767px) 84vw, (max-width: 1279px) calc((100vw - 96px) * 0.47), (max-width: 1695px) calc((100vw - 160px) * .22735), 350px"
                        width="640"
                        class="aspect-video h-auto w-full"
                      />
                    </div>
                    <div
                      class="flex flex-col bg-default px-6 pb-4 pt-6 desktop:pl-[12%] desktop:pr-[14%]"
                    >
                      <a
                        class="mb-3 block text-xl font-medium leading-6 text-default before:absolute before:inset-x-0 before:top-0 before:aspect-video before:bg-[#fff] before:opacity-15 hover:text-accent hover:before:opacity-0"
                        href="/reviews/the-man-with-nine-lives-1940/"
                        >The Man with Nine Lives<!-- --> <span
                          class="text-sm font-normal leading-none text-muted"
                          >1940</span
                        ></a
                      ><picture
                        ><source
                          media="(prefers-color-scheme: dark)"
                          srcset="/svg/2-half-stars-dark.svg" />
                        <img
                          alt="C-: 2.5 stars (out of 5)"
                          class="mb-4"
                          height="18"
                          src="/svg/2-half-stars.svg"
                          width="90"
                      /></picture>
                      <div
                        class="rendered-markdown mb-8 text-base leading-[1.6] tracking-prose text-muted"
                      >
                        <p>
                          Despite Boris Karloff playing a doctor with a secret
                          underground laboratory where he holds six people
                          hostage as unwilling test subjects,
                          <em>The Man with Nine Lives</em> isn’t a mad scientist
                          picture. It’s a drama centered on an ethical dilemma:
                          Would you risk the deaths of six people to save
                          millions?
                          <a
                            class="!no-underline uppercase whitespace-nowrap font-normal font-sans text-accent text-xs leading-none hover:!underline"
                            href="/reviews/the-man-with-nine-lives-1940/"
                            >Continue reading...</a
                          >
                        </p>
                      </div>
                      <div
                        class="font-sans text-xxs font-light leading-4 tracking-wider text-subtle desktop:tracking-wide"
                      >
                        <span>Horror</span
                        ><span>
                          |
                          <!-- -->Mystery</span
                        ><span>
                          |
                          <!-- -->Sci-Fi</span
                        >
                      </div>
                    </div>
                  </li>
                  <li class="relative w-full tablet:w-[47%]">
                    <div class="block">
                      <img
                        src="/_image/?href=%2Fcontent%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=640&amp;h=360&amp;q=80&amp;f=avif"
                        srcset="
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=160&amp;h=90&amp;q=80&amp;f=avif    160w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=320&amp;h=180&amp;q=80&amp;f=avif   320w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=640&amp;h=360&amp;q=80&amp;f=avif   640w,
                          /_image/?href=%2Fcontent%2Fassets%2Fstills%2Fburial-ground-the-nights-of-terror-1981.png%3ForigWidth%3D2000%26origHeight%3D1125%26origFormat%3Dpng&amp;w=1280&amp;h=720&amp;q=80&amp;f=avif 1280w
                        "
                        alt=""
                        decoding="async"
                        loading="lazy"
                        height="360"
                        sizes="(max-width: 767px) 84vw, (max-width: 1279px) calc((100vw - 96px) * 0.47), (max-width: 1695px) calc((100vw - 160px) * .22735), 350px"
                        width="640"
                        class="aspect-video h-auto w-full"
                      />
                    </div>
                    <div
                      class="flex flex-col bg-default px-6 pb-4 pt-6 desktop:pl-[12%] desktop:pr-[14%]"
                    >
                      <a
                        class="mb-3 block text-xl font-medium leading-6 text-default before:absolute before:inset-x-0 before:top-0 before:aspect-video before:bg-[#fff] before:opacity-15 hover:text-accent hover:before:opacity-0"
                        href="/reviews/burial-ground-the-nights-of-terror-1981/"
                        >Burial Ground: The Nights of Terror<!-- --> <span
                          class="text-sm font-normal leading-none text-muted"
                          >1981</span
                        ></a
                      ><picture
                        ><source
                          media="(prefers-color-scheme: dark)"
                          srcset="/svg/3-stars-dark.svg" />
                        <img
                          alt="C: 3 stars (out of 5)"
                          class="mb-4"
                          height="18"
                          src="/svg/3-stars.svg"
                          width="90"
                      /></picture>
                      <div
                        class="rendered-markdown mb-8 text-base leading-[1.6] tracking-prose text-muted"
                      >
                        <p>
                          Teamwork wins. That’s the takeaway from
                          <em>Burial Ground</em>, an Italian zombie horror that
                          packs just enough originality to set itself above the
                          mass of
                          <span data-imdb-id="tt0080057"><em>Zombie</em></span>
                          knock-offs.
                          <a
                            class="!no-underline uppercase whitespace-nowrap font-normal font-sans text-accent text-xs leading-none hover:!underline"
                            href="/reviews/burial-ground-the-nights-of-terror-1981/"
                            >Continue reading...</a
                          >
                        </p>
                      </div>
                      <div
                        class="font-sans text-xxs font-light leading-4 tracking-wider text-subtle desktop:tracking-wide"
                      >
                        <span>Horror</span>
                      </div>
                    </div>
                  </li>
                </ul>
              </div>
            </nav>
          </div>
        </main>
        <footer
          class="flex flex-wrap items-start justify-between gap-[10%] bg-footer px-container py-20 text-inverse tablet:px-12 tablet:pt-10 desktop:p-20"
        >
          <div class="flex flex-col pb-12">
            <div class="flex flex-col">
              <div
                class="whitespace-nowrap font-normal leading-8"
                style="font-size: 1.5625rem"
              >
                <a href="/">Frank&#x27;s Movie Log</a>
              </div>
              <p class="w-full pl-px text-sm italic leading-4 opacity-85">
                by Frank Showalter
              </p>
            </div>
          </div>
          <div
            class="flex flex-col gap-20 pb-20 pt-10 tablet:basis-1/2 tablet:pt-20"
          >
            <ul class="flex w-full flex-col gap-y-10 text-inverse max:w-auto">
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a class="hover:text-accent" href="/">Home</a>
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a class="hover:text-accent" href="/how-i-grade/"
                  >How I Grade</a
                >
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a class="hover:text-accent" href="/reviews/">Reviews</a>
                <ol class="mt-4">
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wide text-inverse-subtle last:mb-0"
                  >
                    <a class="hover:text-inverse" href="/reviews/overrated/"
                      >Overrated</a
                    >
                  </li>
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wide text-inverse-subtle last:mb-0"
                  >
                    <a class="hover:text-inverse" href="/reviews/underrated/"
                      >Underrated</a
                    >
                  </li>
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wide text-inverse-subtle last:mb-0"
                  >
                    <a class="hover:text-inverse" href="/reviews/underseen/"
                      >Underseen</a
                    >
                  </li>
                </ol>
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a class="hover:text-accent" href="/viewings/">Viewing Log</a>
                <ol class="mt-4">
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wide text-inverse-subtle last:mb-0"
                  >
                    <a class="hover:text-inverse" href="/viewings/stats/"
                      >Stats</a
                    >
                  </li>
                </ol>
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a class="hover:text-accent" href="/cast-and-crew/"
                  >Cast &amp; Crew</a
                >
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a class="hover:text-accent" href="/collections/"
                  >Collections</a
                >
              </li>
              <li class="block w-1/2 whitespace-nowrap text-2xl">
                <a class="hover:text-accent" href="/watchlist/">Watchlist</a>
                <ol class="mt-4">
                  <li
                    class="mb-4 ml-1 font-sans text-xs uppercase tracking-wide text-inverse-subtle last:mb-0"
                  >
                    <a class="hover:text-inverse" href="/watchlist/progress/"
                      >Progress</a
                    >
                  </li>
                </ol>
              </li>
            </ul>
            <a
              class="mb-8 w-full max-w-button bg-canvas py-5 text-center font-sans text-xs uppercase tracking-wide text-default hover:bg-inverse hover:text-inverse"
              href="#top"
              >To the top</a
            >
            <p class="w-full font-normal leading-4 text-inverse-subtle">
              All reviews by Frank Showalter. All images used in accordance with
              the<!-- -->
              <a
                class="text-inherit underline decoration-2 underline-offset-4 hover:bg-default hover:text-default"
                href="http://www.copyright.gov/title17/92chap1.html#107"
                >Fair Use Law.</a
              >
            </p>
          </div>
        </footer>
      </div>
    </div>
    <dialog
      aria-label="Search"
      class="fixed left-4 right-4 h-screen w-full rounded-lg border border-default bg-default shadow-lg backdrop:bg-[#000] backdrop:opacity-40 open:flex open:flex-col tablet:mx-auto tablet:mt-16 tablet:h-min tablet:max-h-[min(100vh_-_128px,900px)] tablet:min-h-60 tablet:max-w-[min(100vw_-_64px,36rem)] tablet:rounded-xl desktop:min-h-0"
    >
      <div class="contents" data-dialog-frame>
        <button
          class="px-2 font-mono text-xs font-light hover:bg-inverse hover:text-inverse focus:outline-accent"
          data-close-modal
        >
          ESC
        </button>
        <div class="contents"><div id="pagefind__search"></div></div>
        <div
          class="hidden px-[var(--search-dialog-padding)] py-4 font-sans text-xs font-light tracking-wide text-subtle desktop:block"
        >
          <kbd class="rounded-md bg-default p-2 font-mono text-xs text-subtle"
            >Tab</kbd
          >
          to navigate
          <kbd
            class="ml-4 rounded-md bg-default p-2 font-mono text-xs text-subtle"
            >ESC</kbd
          >
          to close
        </div>
      </div>
    </dialog>
    <script>
      (() => {
        const openBtn = document.querySelector("button[data-open-modal]");
        if (!openBtn) return;
        if (/(Mac|iPhone|iPod|iPad)/i.test(navigator.platform)) {
          openBtn.setAttribute("aria-keyshortcuts", "Meta+K");
          openBtn.setAttribute("title", `Search: ⌘K`);
        }
      })();
    </script>
    <script
      type="module"
      src="/src/layouts/Layout.astro?astro&type=script&index=0&lang.ts"
    ></script>
  </body>
</html>
